scipy/signal/signaltools.py

```diff
@@ -2024,6 +2074,110 @@ def filtfilt(b, a, x, axis=-1, padtype='odd', padlen=None):
     return y
 
 
+def sosfilt(sos, x, axis=-1, zi=None):
+    """
+    Filter data along one dimension using cascaded second-order sections
+
+    Filter a data sequence, `x`, using a digital IIR filter defined by
+    `sos`. This is implemented by performing `lfilter` for each
+    second-order section.  See `lfilter` for details.
+
+    Parameters
+    ----------
+    sos : array_like
+        Array of second-order filter coefficients, must have shape
+        ``(n_sections, 6)``. Each row corresponds to a second-order
+        section, with the first three columns providing the numerator
```

Comment:
I don't think it's too crucial for now because I'd expect 95+% of people who use this function will do so after using an `output='sos'` or `zpk2sos` operation. Feel free to open a PR into my branch, or a subsequent PR if you want to add the docs, though.